whats-on/terraform/services.tf

```diff
@@ -0,0 +1,34 @@
+module "whats_on" {
+  source             = "git::https://github.com/wellcometrust/terraform.git//services?ref=v1.0.4"
+  name               = "whats_on"
+  cluster_id         = "${local.cluster_name}"
+  task_role_arn      = "${module.ecs_whats_on_iam.task_role_arn}"
+  template_name      = "default"
+  vpc_id             = "${local.vpc_id}"
+  nginx_uri          = "wellcome/wellcomecollection_whats_on_nginx:${var.container_tag}"
+  app_uri            = "wellcome/wellcomecollection_whats_on_webapp:${var.container_tag}"
```

Comment:
You'll need to be sure to create these in docker hub, and also allow write access to Circle.